WasteWar/Assets/Scripts/Camera Controls/Zoom.cs

```diff
@@ -0,0 +1,96 @@
+﻿using UnityEngine;
+
+public class Zoom : MonoBehaviour
+{
+    [SerializeField]
+    private Transform route;
+    [SerializeField]
+    private Transform cam;
+    [SerializeField]
+    private Transform rotationRoute;
+
+    //parameter for parametric equation
+    public float t = 0;
+
+    //points along Bezier curve for smooth movement of camera
+    private Vector3 p1;
```

Comment:
meaningful names please

Reply:
gotta disagree on this one, having long variable names in math equations can lead to it being harder understand than having short names + comment above, if this was general purpose code for someone that knows 0 math then sure, but in our scenario i believe it's unnecessary renaming this one when there's a comment explaining what it is, or a rename could be done but the name has to be very short, which i can't think of right now